proposals/0383-allow-let-property-wrapper.md

```diff
@@ -154,6 +128,38 @@ struct ContentView: View {
 
 Similarly, other SwiftUI property wrappers could be `let` declared when they do not require more than a single initialization.
 
+### Property wrappers with `nonmutating set`
+
+Property wrappers that have a `wrappedValue` property with a `nonmutating set` (e.g., SwiftUI's [`@State`](https://developer.apple.com/documentation/swiftui/state/wrappedvalue) and [`@Binding`](https://developer.apple.com/documentation/swiftui/binding/wrappedvalue)) will preserve the reference semantics of the `wrappedValue` implementation even when marked as a `let` declaration. For example:  
+
+```
+@State let weekday: String = "Monday"
+```
+Here, `weekday` is an immutable instance of the `@State` property wrapper, but its `wrappedValue` storage will retain its mutability and reference type traits. This will translate to:
+```
+private let _weekday: State<String> = State<String>(wrappedValue: "Monday")
+var weekday : String {
+  get { 
+    return _weekday.wrappedValue 
+  }
+  nonmutating set {
+    self._weekday.wrappedValue = value
+  }
+  nonmutating _modify { 
+    yield () 
+  }
+}
+var $weekday: Binding<String> {
+  get { return _weekday.projectedValue }
+}
+```
+
+Marking `weekday` as a let declared property will not remove access to its `wrappedValue`'s nonmutating set and `wrappedValue` can be assigned via the backing property:
```

Comment:
To clarify, _only_ via the backing property, and _not_ directly by assigning to `weekday`, correct? The text should make this clear.

Reply:
Yes. I will make this clearer.